src/test/java/com/EighthLight/app/TestGame.java

```diff
@@ -1,75 +1,351 @@
 package com.EighthLight.app;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-
 import java.util.ArrayList;
 import java.util.Arrays;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 class gameTests {
+    private MockPlayer playerOne;
+    private MockPlayer playerTwo;
+    private String correctInput;
+    private String incorrectInput;
+    private ArrayList boardState;
+    private ArrayList gameIsTiedValues;
+    private ArrayList aPlayerWonValues;
+
+    @BeforeEach
+    private void setUp() {
+        playerOne = new MockPlayer("X");
+        playerTwo = new MockPlayer("O");
+        correctInput = Constants.CORRECT_INPUT;
+        incorrectInput = Constants.INCORRECT_INPUT;
+        boardState = new ArrayList();
+    }
 
     @Test
-    void whenStartIsCalledPresentBoardIsCalledWithCurrentBoardWhenGameIsTied() {
-        String userInput = "8";
-        MockUi ui = new MockUi(userInput);
-        ArrayList currentBoard = new ArrayList(Arrays.asList("O", "X", "O",
-                                                             "O", "X", "X",
-                                                             "X", "O", 8));
-
-        ArrayList updatedBoard = new ArrayList(Arrays.asList("O", "X", "O",
-                                                             "O", "X", "X",
-                                                             "X", "O", "X"));
-
-        Player playerOne = new Player("X");
-        Player playerTwo = new Player("O");
-        // MockBoard board = new MockBoard(3, currentBoard, true);
-        Board board = new Board(3);
-        board.createBoard();
-        board.setSpaces(currentBoard);
+    void startGameTiedWithCorrectInput() {
```

Comment:
I think the comment Bjorn made yesterday was a good one. Is there a way to better describe what this test is testing?